### PR TITLE
SISRP-28447 Remove CalCentral Release Notes dependency on ETS website

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.berkeley.edu/browse/SISRP-28447

Please don't merge until we have OK from Ben, @nkutub , and CC QA for this to go into GL8.

### CODE REVIEW

File: app/models/service_alerts/merged.rb [hunk 1]
```diff
@@ -5,7 +5,11 @@ class Merged
 
     def get_feed_internal
       feed = {}
-      feed[:releaseNote] = EtsBlog::ReleaseNotes.new.get_latest
+      feed[:releaseNote] = if (latest_splash = ServiceAlerts::Alert.get_latest_splash)
```
Comment (johncrossman) [inline app/models/service_alerts/merged.rb]: @raydavis this really should be named `last_splash` #the-breeders  ...Okay, nevermind.

**Comment by johncrossman** [comment]
Looks good, @raydavis  Waiting on T

File: app/models/service_alerts/merged.rb (hunk 1 above)
Comment (redconfetti) [inline app/models/service_alerts/merged.rb]: > damn shame.... damn shaaame

**Comment by johncrossman** [comment]
On second thought, I'll wait for "OK from Ben" as description asks.

File: db/schema.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170228210646) do
+ActiveRecord::Schema.define(version: 20170307233607) do
```
Comment (redconfetti) [inline db/schema.rb]: @lyttam - FYI this PR includes a migration, so the release management instructions on migrations may apply with CLC v88

**Comment by redconfetti** [comment]
@raydavis - Oh. So we're not getting rid of ETS Release notes altogether, just implementing a CalCentral-only release note mechanism.

File: db/schema.rb (hunk 2 above)
Comment (raydavis) [inline db/schema.rb]: BTW, @redconfetti , the next release already has some other DB migrations.

**Comment by raydavis** [comment]
@redconfetti , you would be getting rid of ETS Release notes whenever you want to. But I wanted to allow a conservative fallback for the first release at least. Does that make sense?

**Comment by raydavis** [comment]
@nkutub & Janhavi have OKed this for GL8. Thank you both for checking!

**Pull request merged**